ara/utils.py

```diff
@@ -51,6 +53,27 @@ def jinja_pick_status(row):
     return 'ok'
 
 
+@app.template_filter('pathtruncate')
```

Comment:
I see what you're trying to do and I think it's definitely a better idea than what I had in mind. There are some odd results here, though, some playbooks are just showing as '...' and everything has a slash suffix (I guess because of the os.path.join?)
We should probably not truncate further than the basename (i.e, at least keep the basename).

![Alt](http://i.imgur.com/AJnad2H.png)
![Alt](http://i.imgur.com/DmeEuvj.png)